### ISSUE TITLE
Removing loot pool works, but throws error message

### ISSUE THREAD

**Issue by WesCook** [Leviathan143/LootTweaker#7]
I'm playing around with LootTweaker and trying to tweak some recipes for my modpack.  Currently I'm having an issue when trying to remove a loot pool, where it still shows an error message despite working.

The code is as follows:

```
import loottweaker.vanilla.loot.LootTables;
import loottweaker.vanilla.loot.LootTable;
import loottweaker.vanilla.loot.LootPool;

val witherSkeletonTable = LootTables.getTable("minecraft:entities/wither_skeleton"); // Get table
witherSkeletonTable.removePool("pool2"); // Remove original skull pool
```

And the message it prints to chat is: `No loot pool with name pool2 exists!`

However, the code still works.  Checking the entity dump shows that the pool has been removed.

I don't believe this message should show in chat if it's working correctly.

**Comment by Leviathan143** [comment]
The bug is fixed now, it'll be in the next release which should be soonTM. Thanks for the report and good luck with your pack!

**Comment by WesCook** [comment]
Thanks for the quick fix!  I'm finding the 1.10 loot table system a little confusing, but ultimately very powerful.  Excited by some of the possibilities this opens up.

Cheers!
